packages/amplify-cli-npm/binary.ts

```diff
@@ -75,6 +75,7 @@ const getCompressedBinaryUrl = (): string => {
   if (process.env.IS_AMPLIFY_CI && process.env.CODEBUILD_SRC_DIR) {
     // use cloudfront distribution for e2e
     url = `https://${process.env.PKG_CLI_CLOUDFRONT_URL}/${version}/${compressedBinaryName}`;
+    url = url.replace('.tgz', `-${getCommitHash()}.tgz`);
```

Comment:
why do we have to use `process.env.PKG_CLI_CLOUDFRONT_URL` on codebuild ?

Reply:
its dynamic, based on environment (local/beta/prod)